WcaOnRails/app/views/static_pages/faq.html.erb

```diff
@@ -1,5 +1,32 @@
-<% provide(:title, 'Frequently Asked Questions') %>
-
+<% provide(:title, t('faq.title')) %>
+
+<%
+faq_variables = {
+  profile_edit_path: profile_edit_path,
+  current_user_status: if current_user
+                         t('faq.already_logged_in')
+                       else
+                         t('faq.create_an_account', here: link_to(t('common.here'), new_user_registration_path))
+                       end,
+  current_user_claim: if current_user
```

Comment:
I actually wrote it your way first, but I liked the fact that there was one single assignment for each entry in the hash.
I'll just let it as it is :p